mscore/musescore.cpp

```diff
@@ -1198,8 +1213,7 @@ MuseScore::MuseScore()
       QAction* a;
 #ifdef HAS_MIDI
       a  = getAction("midi-on");
-      a->setEnabled(preferences.getBool(PREF_IO_MIDI_ENABLEINPUT));
-      a->setChecked(_midiinEnabled);
+      a->setChecked(preferences.getBool(PREF_IO_MIDI_ENABLEINPUT));
```

Comment:
good change, see also #5378, which did the same thing in a different spot (not sure why I missed this here)

Reply:
It's very easy to miss because the name of the preference has the word “enable” in it, so it tricks the brain into thinking we should be calling `setEnabled` instead of `setChecked`.